src/AudioVideoPlayback/widget/AudioVideoPlayback.js

```diff
@@ -249,6 +259,5 @@ define([
         }
     });
 });
-require(["AudioVideoPlayback/widget/AudioVideoPlayback"], function () {
-    "use strict";
-});
\ No newline at end of file
+
+require(["AudioVideoPlayback/widget/AudioVideoPlayback"]);
```

Comment:
Add spacing to square brackets ie
`require([ "AudioVideoPlayback/widget/AudioVideoPlayback" ])` is easier to read